.gitignore

```diff
@@ -0,0 +1,9 @@
+Packages
+.build
+.DS_Store
+*.xcodeproj
+DerivedData/
+Package.resolved
```

Comment:
DO we want to ignore Package.resolved?  It could mean we're working on different version so Vapor or Packages, which could cause issues.

Reply:
happy to leave it in, i lifted the gitignore from here: https://github.com/vapor/vapor/blob/main/.gitignore 🤣